CommunicationSeminar/settings/production.py

```diff
@@ -146,7 +146,7 @@ def env_get_bool(name, default=None):
             'PORT': '3306',
             'NAME': 'CommunicationSeminarDjango',
             'USER': 'root',
-            'PASSWORD': '2017%ComSem',
+            'PASSWORD': 'root',
```

Comment:
Maybe we could do this so that people can set their own local db passwords to whatever they like?
If we commit this, you'd just need to set an env variable called DB_PASSWORD to "root"
```suggestion
            'PASSWORD': env_get('DB_PASSWORD', '2017%ComSem')
```